frontend/src/pages/Snapshots.tsx

```
@@ -0,0 +1,77 @@
+      </div>
+    )
+  }
+
+  return (
+    <div className="mt-16 min-h-screen p-8 text-gray-600 dark:bg-[#212529] dark:text-gray-300">
+      <div className="mt-16 flex min-h-screen w-full flex-col items-center justify-normal p-5 text-text">
+        <div className="grid grid-cols-1 gap-6 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4">
+          {snapshots &&
+            snapshots.map((snapshot: Snapshots) => (
+              <div key={snapshot.key}>{renderSnapshotCard(snapshot)}</div>
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)